source/administration/changelog.md

```diff
@@ -4,6 +4,152 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Added ability for users to load more recent messages after jumping to a search result.
+ - Cursor now returns to the reply thread input box after deleting a reply on the right-hand sidebar.
+
+#### Performance
+ - Decreased channel load time by optimizing database queries used to fetch threads and parent posts in a channel.
+ - Decreased load time of large channels with 5,000+ messages by up to 90% by optimizing many client functions related to rendering posts and threads.
+ 
+#### Plugins (Beta)
+ - Plugins now have more flexibility to format text, emojis and Markdown.
+ - Added support for plugins to add actions to the sidebar dropdowns.
+ 
+#### Administration
+ - Added support for OWA SMTP servers by supporting the LOGIN authentication method for SMTP.
+ - Added support for AWS Identity and Access Management (IAM) roles for Amazon S3 file storage.
+ - Added a "Test Connection" button to test Amazon S3 connection.
+ - Added a `platform user email` command to change a user's email address.
+
+#### Enterprise Edition
+ - When `ExperimentalTownSquareIsReadOnly` is set to `true`, non-admins can no longer react to messages, pin messages or update channel information.
+ - Added cache invalidation totals to [Performance Monitoring](https://docs.mattermost.com/deployment/metrics.html).
+
+### Bug Fixes
+
+ - Fixed 404 errors about custom emoji in the logs.
+ - Fixed an issue where cursor jumped to end of line when trying to edit text in the middle of search bar.
+ - Fixed an issue where a download link opened images in a new tab instead of downloading them.
+ - Fixed push notifications being sent for your own direct messages.
+ - Fixed plugin webapp manifest field 'bundle_path' to match documentation and be the path to the plugin's JavaScript bundle relative to the root of the plugin, similar to the server manifest 'executable' field.
+ - Fixed an issue where Direct Message channel with yourself did not show up in channel switcher.
+ - Fixed an issue where deleting one username from "add member to a channel" field deleted all names.
+ - Fixed an issue where link was missing in an ephemeral message to add mentioned user to the channel.
+ - Fixed an issue where View/Manage members should have been sorted by username, not online status.
+ - Fixed an issue where a non-system-admin should not see `Is Trusted` option on OAuth 2.0 integrations.
+ - Fixed an issue with being unable to click on pinned post, channel members, and so on with keyboard focus on search box.
+ - Fixed an issue where Mattermost only imported first user during Slack import.
+ - Fixed an issue where cleared search term reappeared after closing RHS.
+ - Fixed an issue where image was posted while RHS was open with an oversized thumbnail.
+ - Fixed an issue where both "test" and "test.example" were added to channel when adding "test.example" user to a channel via ephemeral message that appears when mentioning someone not in a channel.
```

Comment:
Example is too specific. Propose:

Fixed an issue with adding users to channels when the usernames contained periods.